cli/backends.go

```diff
@@ -342,6 +342,21 @@ func newRepo(conn *pgxpool.Pool, personService backends.PersonService, organizat
 			"set_status":               mutate.SetStatus,
 			"set_locked":               mutate.SetLocked,
 		},
+
+		CandidateRecordLoaders: []repositories.CandidateRecordVisitor{
+			func(c *models.CandidateRecord) error {
+				if c.StatusPersonID != nil && *c.StatusPersonID != "" {
+					person, err := personService.GetPerson(*c.StatusPersonID)
+					if err != nil {
+						logger.Warn("error loading status person in candidate record", "statuPersonID", *c.StatusPersonID, "id", c.ID, "error", err)
+						c.StatusPerson = backends.NewDummyPerson(*c.StatusPersonID)
+					} else {
+						c.StatusPerson = person
+					}
+				}
+				return nil
+			},
+		},
```

Comment:
After calling this new CandidateRecordLoader, we always also call the unmarshall on the metadata field and then all PublicationLoaders on `rec.Publication`. I was looking to add those two routines as other CandidateRecordLoaders so this can all be done from one loop, but that didn't work since the PublicationLoader needs access to `Repo.config.PublicationLoaders` which isn't available from another loader I guess.

The unmarshall would be possible as a separate CandidateRecordLoader but not sure if that makes sense if doing the next step is still implemented in each repo method. Better to figure out a way to do it all as CandidateRecordLoaders.